updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0
+
 :: Update hosts file
-python updateHostsFile.py -a
+if exist updateHostsFile.exe (
+    echo run exe
+    updateHostsFile --auto --extensions fakenews social gambling

```

Comment:
Where does `updateHostsFile` come from ? Is it specific to your environment/scope?

Reply:
In this case `updateHostFile` is `updateHostsFile.exe` and the extension just isn't required.

Reply:
@evenprimes :+1: